DEVELOP.md

```diff
@@ -0,0 +1,254 @@
+# Chaos Mesh Development Guide
+Chaos Mesh currently have many types of chaos such as IOChaos, NetworkChaos, PodChaos. This guide aims to introduce the way to add new chaos type for Chaos Mesh, which includes setting up the development environment and adding a hello-world chaos type.
+
+## Setting Up The Development Environment
+### Requirements
+You should have these components installed in your system:
+- golang (>= v1.13), if your golang version < v1.13, you can use golang version manager such as [gvm](https://github.com/moovweb/gvm) to switch to a newer one.
+- [yarn](https://yarnpkg.com/lang/en/) and [nodejs](https://nodejs.org/en/): for chaos-dashboard
+- docker
+- gcc
+- [kind](https://github.com/kubernetes-sigs/kind)
+
+### Step By Step
+Firstly, clone the repo
+```
+git clone https://github.com/pingcap/chaos-mesh.git
+cd chaos-mesh
+```
+Then, install [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) and [kustomize](https://github.com/kubernetes-sigs/kustomize)
+```
+make install-kubebuilder
+make install-kustomize
+```
+After this, you should add kubebuilder and kustomize to your PATH environment:
```

Comment:
this is not necessary, because we did it in `make install-kubebuilder`  https://github.com/pingcap/chaos-mesh/blob/master/Makefile#L161-L183